src/app/componments/header/LocaleButton.styles.ts

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { Theme } from '../../../assets/styles/theme.types';
+
+export const LocaleButton = styled.button<{ theme: Theme }>`
```

Comment:
Jeśli masz kilka razy wyciągnięcie różnych rzeczy z theme, to można użyć składni funkcyjnej z funkcją `css`: 
```
... = styled.button<{ theme: Theme }>({ theme: { fontSize, color }) => css`
  tutaj wszystkie style
`
```